liquid_node/backup.py

```diff
@@ -19,6 +19,18 @@
 SNOOP_BLOBS_DATA_ALLOC = "hoover-deps:minio-blobs"
 SNOOP_ORIGINAL_DATA_ALLOC = "hoover-deps:minio-collections"
 HTTP_PORT = config.port_lb
+# list of postgres databases that should not be backed up.
+# the format should be <schema>.<table name>
+DATABASES_TO_IGNORE = [
```

Comment:
this file works with 10 databases in 5 services, which one are we talking about?

plz rename const name to something like "SNOOP_PG_TABLES_TO_IGNORE" or something similarly specific